msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md

```diff
@@ -290,6 +299,91 @@ async def _mention_activity(self, turn_context: TurnContext):
+```
+
+##### Mention tags in an Adaptive Card
+
+In the Adaptive Card schema,  under the `mentioned` object, add the `"type": "tag"` property.  If the  `"type": "tag"` property isn't added, the bot treats the mention as a user mention.
+
+You can get the list of the tags available in the channel using the [List teamworkTags](/graph/api/teamworktag-list?view=graph-rest-1.0&tabs=http&preserve-view=true) API.
+
+Example:
+
+```json
+​{ 
+​    "type": "mention", 
+    ​"text": "<at>Test Tag</at>", 
+​    "mentioned": { 
+            ​"id": "base64 encoded id" ,// tag graph 64 base ID
+​            "name": "Test Tag", 
+            ​"type": "tag" 
+​    } 
+​} 
+```
+
+###### Query Parameters
+
+|Name |Description |
+|---------|----------------|
+|`type`| The type of mention. The supported type is `tag`.|
+|`id`|The tag format is base 64 encoded ID​. For example, `NTI4ZGJlM2YtMTVlMC00ZTM3LTg0YTEtMDBjYzMwNTg0N2RkIyNlYzgwMTVmMC1iMmYxLTQxZTItODA0OC1hMGE2OTcwNmM5ZGIjI3RxRE04YndyVQ==​`|
+
+###### Error code
+
+| Status code | Error code | Message values | Retry request | Developer action|
+|----------------|-----------------|-----------------|----------------|----------------|
+| 400 | **Code**: `Bad Request` | ​Mentioned tag with ID {id string} doesn't exist in current team<br/>​Tag can only be mentioned in channel<br/>Invalid mentioned tag because no tag exists in the team| No | Reevaluate request payload for errors. Check returned error message for details. |
+| 502 | **Code**: `Bad Gateway` | Invalid team group ID<br/> ​Malformed tenant ID for the tag<br/> ​Mention ID can't be resolved | No |Retry manually.|
+
+##### Throttling limits
+
+Any request can be evaluated against multiple limits, depending on the scope, the window type (short and long), number of tags per message, and other factors. The first limit to be reached triggers throttling behavior.
+
+Ensure that you don't exceed the throttling limits to avoid failed message delivery. For example, a bot can send only two messages with tags mention in a five-second window and each message can have only up to 10 tags.
+
+The following table lists the throttling limits for tag mentions in a bot:
+
+|​Scope   |​Window Type  |Number of tags per message  |​Time windows (sec)  |​Maximum number of messages per time window  |
+|------------------------|------------|-----------|----------|----------|
+|​Per bot per thread     |   ​Short     |    10     |     5    |     2    |
+| &nbsp;                |   ​Long      |    10     |     60   |     5    |
+|​All bots per thread    |   ​Short     |    10     |     5    |     4    |
+| &nbsp;                |   Long      |    10     |     60   |     5    |
+
+##### Limitations
+
+* Tag mentions are supported only in bot to client message flow with text and Adaptive Card.
+* Tag mentions aren't supported in shared and private channels.
+* Tag mentions aren't supported in connectors.
+* Tag mentions don't support the invoke function in a bot.
```

Comment:
invoke flow

Reply:
Tag mentions don't support the invoke flow in a bot.